### PR TITLE
Fix final_preparation for already normalized data

### PR DESCRIPTION
**Pull request by andbue**
When normalize=False is set, FinalPreparation must not multiply the data with 255. Otherwise it renders many line images unreadable.

### CODE REVIEW

**Comment by ChWick** [comment]
can you verify if my fix works also for you?

**Review by ChWick — APPROVED** [review]

File: calamari_ocr/utils/image.py [hunk 1]
```diff
@@ -95,3 +93,26 @@ def to_uint8(data: np.ndarray) -> np.ndarray:
...
+    """Read an image and returns it as float32
+
+    All arrays are rescaled to the range 0...1
+    """
+    if data.dtype == np.dtype("uint8"):
+        data = data.astype("float32") / 255
+    elif data.dtype == np.dtype("int8"):
+        data = (data.astype("int16") + 128).astype("float32") / 255
+    elif data.dtype == np.dtype("uint16"):
+        data = data.astype("float32") / 256 / 255
```
Comment (andbue) [inline calamari_ocr/utils/image.py:108]: I'm a bit confused by the 256 here: max of uint16 should be 65535, divided by 255 equals 257. It's 35 degrees in here so I might be wrong...

Comment (ChWick) [reply calamari_ocr/utils/image.py]: You are totally right, I wasn't thinking. Fixed it!

Comment (ChWick) [reply calamari_ocr/utils/image.py]: Also for int16 (please verify, 35 seems to be good for your brain :-) )

Comment (andbue) [reply calamari_ocr/utils/image.py]: Looks good now!

**Pull request merged**